src/server/environment.js

```diff
@@ -21,7 +21,9 @@ const IS_DEV = NODE_ENV === 'development'
 
 const LOGIN_EXPIRY_MINUTES = parseInt(process.env.LOGIN_EXPIRY_MINUTES) || 30
 
-const PORT = process.env.PORT || 3000
+const PORT = parseInt(process.env.PORT) || 3000
+const API_DOMAIN = process.env.API_DOMAIN
```

Comment:
we already have a process.env.BASE_URL. i didn't deal with it in https://github.com/usdigitalresponse/arpa-reporter/pull/331 because it's only used client-side (in src/router/index)

but actually, i am not seeing `BASE_URL` set anywhere, including in dev or in the production environments, so i wonder if it can be safely removed